src/popup/ducks/authServices.ts

```diff
@@ -110,7 +110,7 @@ export const confirmPassword = createAsyncThunk<
   }
   if (!res.publicKey) {
     return thunkApi.rejectWithValue({
-      errorMessage: "The password you entered is incorrect",
+      errorMessage: "Incorrect Password",
```

Comment:
Updated the copy to match [this](https://projects.invisionapp.com/prototype/Lyra-Lite-ck60vjr6j003ky301z14bfmta/play/a856ab00)